docs/resources/projects/settings/general-settings.md

```diff
@@ -0,0 +1,195 @@
+---
+slug: general-settings
+title: General Settings
+tags: [project, assets, splash screen, launcher icon, adaptive icon]
+keywords: [project name, package name, asset, download settings, assets, splash screen, launcher icon, adaptive icon]
+description: Learn how to configure general settings for your FlutterFlow app.
+sidebar_position: 0
+---
+
+# General Settings
+This section serves as the control center for configuring essential aspects of your app.
+
+
+## App Details
+This section allows you to customize a wide range of settings that are crucial for the overall functionality and appearance of your application. From basic configurations like project and package (bundle) name to more advanced settings like theme customization and navigation control.
+
+### Change Project, Package, and Display Name
+
+If you didn't decide on the app or package name when creating a new project, you might have entered a random name. From here, you can change the name anytime.
+
+You can set a different name for the *App Store* and *Play Store* using the **Display Name**.
+
+:::tip
+After changing the package name, errors may appear on the toolbar due to invalidated Firebase config files. To resolve this, generate new config files by going to **Settings & Integrations > Project Setup > Firebase > Regenerate Config Files**.
+:::
+---
```

Comment:
We dont need any divider in this page. Please remove them throughout the page.

Reply:
Removed.